ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/config/tls/client/PicoCliClientTlsOptions.java

```diff
@@ -25,24 +27,41 @@
   @SuppressWarnings("UnusedVariable")
   @Option(
       names = "--downstream-http-tls-enabled",
-      description = "Flag to enable TLS connection to web3 provider. Defaults to disabled",
+      description = "Flag to enable TLS connection to web3 provider. Defaults to disabled.",
       arity = "0",
       required = true)
   private boolean tlsEnabled = false;
 
   @ArgGroup(exclusive = false)
   private PicoCliKeyStoreOptions keyStoreOptions;
 
-  @ArgGroup(exclusive = false)
-  private PicoCliClientTlsTrustOptions trustOptions;
+  @Option(
+      names = "--downstream-http-tls-known-servers-file",
+      description =
+          "Path to a file containing the hostname, port and certificate fingerprints of web3 providers to trust. Must be specified if CA auth is disabled.",
+      paramLabel = MANDATORY_FILE_FORMAT_HELP,
+      arity = "1")
+  private Path knownServersFile;
+
+  @Option(
+      names = "--downstream-http-tls-ca-auth-enabled",
+      description =
+          "If set, will use the system's CA to validate received server certificates. Defaults to enabled.",
+      arity = "1")
+  private boolean caAuthEnabled = true;
 
   @Override
   public Optional<KeyStoreOptions> getKeyStoreOptions() {
     return Optional.ofNullable(keyStoreOptions);
   }
 
   @Override
-  public Optional<ClientTlsTrustOptions> getTrustOptions() {
-    return Optional.ofNullable(trustOptions);
+  public boolean isCaAuthEnabled() {
+    return caAuthEnabled;
+  }
+
+  @Override
+  public Path getKnownServersFile() {
+    return knownServersFile;
```

Comment:
You can return Optional<Path> from this method, then you don't need to worry about checking for nulls. i.e. `return Optional.ofNullable(knownServersFile)`

Reply:
done